k8s/pricing-deploy.yaml

```diff
@@ -33,7 +33,7 @@ spec:
       terminationGracePeriodSeconds: 35
       containers:
       - name: pricing
-        image: akka.cqrs.pricing:0.2.0
+        image: akka.cqrs.pricing:0.2.1
```

Comment:
These versions should be kept at 2.0, it is actually an exercise for the students

Reply:
But in dev branch, all should work out of the box, right? And it is 0.2.1 set in RELEASE_NOTES file, so `build Docker phobos` builds 0.2.1 images. That's why going to update this - need to make dev branch working out of the box. Just like `master` branch should.

Then we will propagate changes to `lesson*` branches, and will need to cut off some of my changes from there.